manifests/charts/istio-control/istio-discovery/files/injection-template.yaml

```diff
@@ -84,8 +84,16 @@ template: |
     {{- end }}
   {{- end }}
     securityContext:
-      allowPrivilegeEscalation: {{ .Values.global.proxy.privileged }}
-      privileged: {{ .Values.global.proxy.privileged }}
+    {{- if .Values.global.proxy.privileged }}
+      allowPrivilegeEscalation: true
+      privileged: true
+    {{- else if and (eq (annotation .ObjectMeta `sidecar.istio.io/interceptionMode` .ProxyConfig.InterceptionMode) `TPROXY`) (not .Values.istio_cni.enabled)}}
```

Comment:
What happens if CNI is enabled ? Don't you still need NET_ADMIN for original src to work ?

Reply:
Also: it is simpler to just document that for 'original src' user must set proxy.priviledged. It's also clear to the user.

Reply:
If we remove this chart change, then when user has installed the istio with non-privileged mode, it could not use original src feature later when they need to deploy app.

That's my concern.

Reply:
@costinm Does this make sense